isolation/_02_RepeatableRead/foreign_key_column/answer/delete_truncate_01.answer

```diff
@@ -7,8 +7,6 @@
 |    1    'a'  
```

Comment:
The dont_reuse_oid version of `isolation/_02_RepeatableRead/foreign_key_column/answer/delete_truncate_01.answer` is needed